frontend/src/features/fields/fieldService.tsx

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+
+const API_URL = '/api/fields/';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const SERVER_URL = process.env.REACT_APP_SERVER_URL || '';
+
+interface FieldServiceObject {
+  EntityType: [String];
+  LocationType: [String];
+  CombinedAreaName: [String];
+  OrgStructure: [String];
+  DemographicsServed: [String];
+  FoodDistModel: [String];
+  FoodTypes: [String];
+}
+
+const getFields = async (token: string) => {
+  const config = {
+    headers: {
+      Authorization: `Bearer ${token}`
+    }
+  };
+  const response = await axios.get(API_URL, config);
+  return response.data;
+};
+
+const getFieldByName = async (fieldName: string, token: string) => {
+  const config = {
+    headers: {
+      Authorization: `Bearer ${token}`
+    }
+  };
+  const response = await axios.get(`${API_URL}?name=${fieldName}`, config);
```

Comment:
```suggestion
const response = await axios.get(`${API_URL}${fieldName}`, config);
```
Since the route for this is `GET /api/fields/:name`, the correct way to make the call is actually to just add the field name directly after the api route. 

Formatting a request this way is called using 'params', which is generally used when the data passed (fieldName in this case) is small and noncritical. Then the name can be retrieved from the request using `req.params` (which I changed above in fieldControllers)
 
This can be a little confusing, this article explains the concept pretty well https://dev.to/gathoni/express-req-params-req-query-and-req-body-4lpc